finetuner/__init__.py

```diff
@@ -308,6 +308,63 @@ def get_token() -> str:
     return ft.get_token()
 
 
+def build_model(
+    name: str,
+    batch_size: int = 32,
+    select_model: Optional[str] = None,
+    device: str = 'cpu',
+    logging_level: str = 'DEBUG',
+    is_onnx: bool = False,
+    model_options: Dict[str, Any] = {},
+):
+    """
+    Builds a pre-trained model from a given descriptor.
+
+    :param name: Refers to a pre-trained model, see
+        https://finetuner.jina.ai/walkthrough/choose-backbone/ for a list of all
+        supported models
+    :param batch_size: Incoming documents are fed to the graph in batches, both to
+        speed-up inference and avoid memory errors. This argument controls the
+        number of documents that will be put in each batch.
+    :param select_model: Finetuner run artifacts might contain multiple models. In
+        such cases you can select which model to deploy using this argument. For CLIP
+        fine-tuning, you can choose either `clip-vision` or `clip-text`.
+    :param device: Whether to use the CPU, if set to `cuda`, a Nvidia GPU will be used.
+        otherwise use `cpu` to run a cpu job.
+    :param logging_level: The executor logging level. See
+        https://docs.python.org/3/library/logging.html#logging-levels for available
+        options.
+    :param is_onnx: The model output format, either `onnx` or `pt`.
+    :param model_options: A dictionary of model specific options
+    :return: A built pretrained model
+    """
+    from commons.models.inference import (
+        ONNXRuntimeInferenceEngine,
+        TorchInferenceEngine,
+    )
+    from commons.runner.model import RunnerModel
```

Comment:
from finetuner.commons.runner ...

Reply:
Imports in other methods in this file import from commons like this, are those also incorrect?